Documentation/Examples.md

```diff
@@ -71,6 +71,25 @@ label.textColor = R.color.appColors.textColor()
 
 There are some points to keep in mind when using Color palettes, see [About Colors](Colors.md)
 
+## Localized strings
+
+*Vanilla*
+```swift
+let welcomeMessage = NSLocalizedString("welcome.message", comment: "")
+let settingsTitle = NSLocalizedString("title", tableName: "Settings", comment: "")
+
+let welcomeName = String(format: NSLocalizedString("welcome.withName", comment: ""), locale: NSLocale.currentLocale(), "Alice")
```

Comment:
I think the example is better suited by using the `String(format:arguments:)` overload here. That one uses the default locale, prevents confusion about what the R.swift version below does with the locale.